documentation/source/coroutines.rst

```diff
@@ -202,3 +137,50 @@ type of generator function that can be iterated with ``async for``:
 
 More details on this type of generator can be found in :pep:`525`.
 
+
+Generator-based coroutines
```

Comment:
Likely will want a label on this, maybe

```suggestion
.. yield-syntax:

Generator-based coroutines
```

Or whatever the syntax for label markers is.

Reply:
Because it took me a while to figure it out, I'll mention it's:

`.. _ref_name:`

`:ref:'ref_name'`

Note the required underscore and lack of it when using it out.

Reply:
Pretty sure the ref should be with backticks, not quotes.

Reply:
yeah, I just haven't figured out how to next backticks.